dsp/0181.md

```diff
@@ -0,0 +1,63 @@
+---
+DSP: 0181
+Title: Portable Experiences Interface
+Author: World Team
+Status: Draft
+Type: Standards
+Created: 2021-01-19
+---
+
+# Portable experiences interface for the SDK
+
+The interface would be included from the SDK using the identifier
+
+```typescript
+import * as PortableExperiences from '@decentraland/PortableExperiences'
+```
+
+```typescript
+
+ enum ExecutorType {
+  ESCENE = 'ESCENE',
+  WEARABLE = 'WEARABLE',
+  QUEST_UI = 'QUEST_UI'
+}
+
+type Executor = {
+  type: ExecutorType
+  identifier: string
+}
+
+type PortableExperienceIdentifier = string
+
+type PortableExperienceHandle = {
+  pid: string
+  identifier: PortableExperienceIdentifier
+  parentProcess: Executor
+}
+
+export interface IPortableExperiencesController {
+  /**
+   * Starts a portable experience.
+   * @param {Executor} [executor] - The executor that will trigger the PE
+   * @param  {PortableExperienceIdentifier} [peIdentifier] - Information to identify the PE
+   *
+   * Returns the handle of the portable experience.
+   */
+  spawn(executor: Executor, peIdentifier: PortableExperienceIdentifier): Promise<PortableExperienceHandle>
```

Comment:
I think the executor will be set by context. No need to specify "who am I" because the controller will already know

Reply:
`peIdentifier` it would be better to have a single object containing the ID to be able to extend the object in the future, in the contrary, overloading the function to accept different parameters would be more conflictive.

```typescript
type SpawnPortableExperienceParameters = {
  contentIdentifier: PortableExperienceIdentifier
}
```

would suffice